docs/podman.1.md

```diff
@@ -143,10 +143,22 @@ For the annotation conditions, libpod uses any annotations set in the generated
 
 For the bind-mount conditions, only mounts explicitly requested by the caller via `--volume` are considered.  Bind mounts that libpod inserts by default (e.g. `/dev/shm`) are not considered.
 
+Hooks are not used when running in rootless mode.
+
 **registries.conf** (`/etc/containers/registries.conf`)
 
 registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
 
+## Rootless mode
+Podman can also be used as non-root user.  When podman runs in rootless mode, an user namespace is automatically created.
+
+Containers created by a non-root user are not visible to other users and are not seen or managed by podman running as root.
+
+Images are pulled under `XDG_DATA_DIR` when specified, otherwise in the home directory of the user under `.containers/storage`.
```

Comment:
adding @nalind since we are discussing the same thing for Buildah (https://github.com/projectatomic/buildah/pull/823)

Reply:
Just remove XDG_DATA_DIR and put the content in ~/.containers/storage

Reply:
Agreed, $XDG_DATA_DIRS in my current session is set to "/home/nalin/.local/share/flatpak/exports/share/:/var/lib/flatpak/exports/share/:/usr/local/share/:/usr/share", which doesn't look like a set of appropriate locations.

Reply:
> Agreed, $XDG_DATA_DIRS in my current session...

What about `XDG_DATA_HOME`?  This sort of thing seems like a great match for that variable, and it would be nice to support a standard way for users to declare their preference.

Reply:
$XDG_DATA_HOME defines the base directory relative to which user specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.

XDG_DATA_HOME is fine with me.

Reply:
changed to use XDG_DATA_HOME.

I've changed atomic as well to use the same path: https://github.com/projectatomic/atomic/pull/1243